.github/workflows/delivery-chocolatey.yml

```diff
@@ -4,25 +4,49 @@ on:
   release:
     types:
       - released
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: The release tag to distribute
+        required: true
 
 env:
   CHOCO_PATH: chocolatey
 
+defaults:
+  run:
+    shell: bash
```

Comment:
> Was this tested on Windows? IIRC, the default shell would be powershell in which case I'm not sure the run commands would work as expected. (from https://github.com/buildpacks/pack/pull/927/files#r514303529)

@jromero I'm using the default shell set to `bash` over here, which is why it works.